content/en/docs/contributing/policy.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Feature Policy"
+linkTitle: "Feature Policy"
+weight: 70
+type: "docs"
+---
+
+We are open to feature requests and PRs implementing those. If you plan on contributing a new feature into cert-manager we recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
+
+# Features we will not allow
+
+Certain features have ben requested before but are not able to being implemented properly or not able to being implemented without breaking the security model.
```

Comment:
a few typos

Certain features have been requested before, but have not been implemented properly or not able to be implemented without breaking the security model.